src/main/kotlin/no/nav/familie/ks/sak/kjerne/forrigebehandling/EndringIEndretUtbetalingAndelUtil.kt

```diff
@@ -5,9 +5,29 @@ import no.nav.familie.ks.sak.kjerne.endretutbetaling.domene.EndretUtbetalingAnde
 import no.nav.familie.ks.sak.kjerne.endretutbetaling.domene.Årsak
 import no.nav.familie.tidslinje.Tidslinje
 import no.nav.familie.tidslinje.tilTidslinje
+import no.nav.familie.tidslinje.utvidelser.kombiner
 import no.nav.familie.tidslinje.utvidelser.kombinerMed
+import java.time.YearMonth
 
 object EndringIEndretUtbetalingAndelUtil {
+    fun utledEndringstidspunktForEndretUtbetalingAndel(
+        nåværendeEndretAndeler: List<EndretUtbetalingAndel>,
+        forrigeEndretAndeler: List<EndretUtbetalingAndel>,
+    ): YearMonth? {
+        val allePersoner = (nåværendeEndretAndeler.mapNotNull { it.person?.aktør } + forrigeEndretAndeler.mapNotNull { it.person?.aktør }).distinct()
```

Comment:
Kanskje det er greit å splitte dette litt opp? Den linjen ble en liten munnfull. Ingen stor sak da 😄 

```suggestion
        val nåværenderAktører = nåværendeEndretAndeler.mapNotNull { it.person?.aktør }
        val forrigeAktører = forrigeEndretAndeler.mapNotNull { it.person?.aktør }
        val alleAktører = (nåværenderAktører + forrigeAktører).distinct()
```